lti_emailer/canvas_api_client.py

```diff
@@ -43,6 +43,24 @@ def get_sections(canvas_course_id):
     return result
 
 
+def get_sis_course_id(canvas_course_id):
+    """
+    This method will retrieve the sis_course_id given the canvas course id
+    """
+    print ("in get_sis_course_id for canvas_course_id=%s" %canvas_course_id)
+    try:
+        response = courses.get_single_course_courses(SDK_CONTEXT, canvas_course_id, 'all_courses')
+        course = response.json()
+        sis_course_id = course['sis_course_id']
+        logger.debug ("returning sis course id  = %s for canvas course %s" % (sis_course_id, canvas_course_id ))
+    except CanvasAPIError:
+        logger.exception(
+            "Failed to get canvas course information for canvas_course_id %s ",
+            canvas_course_id,
+        )
+        raise
+    return sis_course_id
```

Comment:
It would be good to throw the returned canvas course dict into the cache. Check out the other function in this module.